Packs/MISP/Integrations/MISP_V2/MISP_V2.py

```diff
@@ -1039,7 +1039,7 @@ def add_object(event_id: str, obj: MISPObject):
         errors = extract_error(response["errors"])
         error_string = str()
         for err in errors:
-            error_string += f'' \
+            error_string += '' \
```

Comment:
shouldnt we remove this empty string?

Reply:
In theory, but because I don't know why it's there in the first place and on the _offchance_ that it is purposeful... I decided to leave it.

Reply:
If you want it removed, I'll go ahead and remove it.

Reply:
i think we can remove, dont see why its needed